docs/build-rdoc.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -ex
+pushd ../mlflow/R/mlflow
+Rscript -e 'install.packages("devtools", repos = "https://cloud.r-project.org")'
+Rscript -e 'devtools::install_deps(dependencies = TRUE)'
+rm -rf man
+Rscript -e "roxygen2::roxygenise()"
```

Comment:
Should we also install `roxygen2` here? 

```
The downloaded binary packages are in
        /var/folders/rq/g5vkbkv93r12yb1t374lfj0r0000gn/T//Rtmp0CIm6G/downloaded_packages
+ rm -rf man
+ Rscript -e 'roxygen2::roxygenise()'
Error in loadNamespace(name) : there is no package called ‘roxygen2’
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
make: *** [html] Error 1
```

Reply:
Thanks for the catch, switched this to use `install_dev_deps` which should install `roxygen2`: https://www.rdocumentation.org/packages/devtools/versions/1.13.6/topics/install_deps

Reply:
```
+ Rscript document.R
Error in loadNamespace(name) : there is no package called ‘Rd2md’
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
make: *** [html] Error 1
```